packages/web-components/src/components/ic-dialog/ic-dialog.css

```diff
@@ -0,0 +1,141 @@
+@import "../../global/normalize.css";
+
+/** 
+  * @prop --ic-z-index-dialog: z-index of dialog 
+  */
+
+:host {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: 100% !important;
+  background: rgb(0 0 0 / 60%);
+  z-index: var(--ic-z-index-dialog);
+  opacity: 0;
+  transition: opacity var(--ic-easing-transition-slow);
+}
+
+:host(.fade-in) {
+  opacity: 1;
+  transition: opacity var(--ic-easing-transition-slow);
+}
+
+:host(.hidden) {
+  display: none;
+}
+
+.dialog {
+  background-color: var(--ic-architectural-white);
+  border: 1px solid var(--ic-architectural-300);
+  border-radius: var(--ic-border-radius);
+  padding: var(--ic-space-xs) var(--ic-space-md);
+  transform: translateY(-3rem);
+  transition: transform 1000s;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+:host(.fade-in) .dialog {
+  transform: translateY(0);
+  transition: transform var(--ic-easing-transition-slow);
+}
+
+.small {
+  width: 50%;
+  max-width: 25rem;
+  min-height: 11rem;
+  max-height: 70vh;
+}
+
+.medium {
+  width: 70vw;
+  max-width: 44rem;
+  min-height: 12.5rem;
+  max-height: 70vh;
+}
+
+.large {
+  width: 90vw;
+  max-width: 62.5rem;
+  min-height: 12.5rem;
+  max-height: 90vh;
+}
+
+.heading-area {
+  display: flex;
+}
+
+.close-icon {
+  margin-left: auto;
+}
+
+.close-icon-svg {
+  display: flex;
+  align-items: center;
+}
+
+.content-area {
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  padding: var(--ic-space-xs);
+  margin: calc(-1 * var(--ic-space-xs));
+}
+
+.content-area::-webkit-scrollbar {
+  display: none;
+}
+
+.status-alert {
+  margin-top: var(--ic-space-xs);
+}
+
+.message {
+  margin-top: var(--ic-space-xs);
+}
+
+.dialog-controls {
+  margin-top: auto;
+  margin-bottom: var(--ic-space-xxs);
```

Comment:
Looks like the gap underneath the buttons is slightly larger than the 8px in the designs because of this margin:
<img src="https://user-images.githubusercontent.com/111882109/225660874-c27c65a6-1ed9-46bb-a62c-dc02b8ff2657.png" width="300">